apps/dashboard/pages/index.tsx

```diff
@@ -25,6 +26,7 @@ const Index = () => {
 
   return (
     <AdminLayout seo={{ title: t('home') }}>
+      <SubToNotification />
```

Comment:
Shall we show the component under a feature flag and test it in staging first?

```suggestion
      {process.env.NEXT_PUBLIC_ENABLE_NOTIFICATIONS && <SubToNotification />}
```